image-rs/scripts/build_attestation_agent.sh

```diff
@@ -9,18 +9,26 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-parameters="KBC=offline_fs_kbc"
+parameters=("KBC=offline_fs_kbc")
 
 [ -n "${BASH_VERSION:-}" ] && set -o errtrace
 [ -n "${DEBUG:-}" ] && set -o xtrace
-[ -n "${TTRPC:-}" ] && parameters+=" ttrpc=true"
+if [[ -n "${TTRPC:-}" ]]; then
+    parameters+=("ttrpc=true")
+    dest_dir_suffix="ttrpc"
+else
+    dest_dir_suffix="grpc"
```

Comment:
Will we also need changes to the kata initrd building scripts and peer pods tooling to account for the new location?

Reply:
I'm not sure I understand that question. I think this build script is only ever called by the image-rs tests. Everything else should use the attestation-agents Makefile. The small modification to the attestation-agent Makefile should only change behavior in edge cases.